src/generic_ui/polymer/roster.html

```diff
@@ -57,6 +57,7 @@
       bottom: 122px;
       right: 30px;
       left: 30px;
+      min-height: 218px; /* TODO: keep downArrow and add button anchored to bottom when min-height kicks in. Ideally we'll get this for free with https://github.com/uProxy/uproxy/issues/2547 */
```

Comment:
weird! a6e37c8 bumps the arrow up by 3px for good measure. i think that oughta fix it. thanks for catching!